tests/jeos/firstrun.pm

```diff
@@ -166,6 +166,7 @@ sub run {
 
     # openSUSE JeOS has SWAP mounted as LABEL instead of UUID until kiwi 9.19.0, so tw and Leap 15.2+ are fine
     verify_mounts unless is_leap('<15.2') && is_aarch64;
+    assert_script_run 'cat /proc/cmdline';
```

Comment:
Why do you need this?
If you want to store the info, it is probably better to use `record_info` or upload the result to a file?

Reply:
It was only for my debugging purposes, so I could easily compare the kernel parameters. I will remove it, thanks! :)